core/src/test/java/com/graphhopper/storage/GraphHopperStorageLMTest.java

```diff
@@ -1,73 +0,0 @@
-package com.graphhopper.storage;
-
-import com.graphhopper.GraphHopper;
-import com.graphhopper.config.LMProfile;
-import com.graphhopper.config.Profile;
-import com.graphhopper.reader.ReaderWay;
-import com.graphhopper.routing.ev.Subnetwork;
-import com.graphhopper.routing.util.CarFlagEncoder;
-import com.graphhopper.routing.util.EncodingManager;
-import com.graphhopper.util.GHUtility;
-import com.graphhopper.util.Helper;
-import org.junit.jupiter.api.Test;
-
-import java.io.File;
-
-import static com.graphhopper.util.GHUtility.updateDistancesFor;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
-public class GraphHopperStorageLMTest {
-    @Test
-    public void testLoad() {
-        String defaultGraphLoc = "./target/ghstorage_lm";
-        Helper.removeDir(new File(defaultGraphLoc));
-        CarFlagEncoder carFlagEncoder = new CarFlagEncoder();
-        EncodingManager encodingManager = new EncodingManager.Builder().add(carFlagEncoder).add(Subnetwork.create("my_profile")).build();
-        GraphHopperStorage graph = GraphBuilder.start(encodingManager).setRAM(defaultGraphLoc, true).create();
```

Comment:
Also here we use a combination of low- and high-level API. This test does no longer test anything useful so I deleted the entire class.